pages/admin/index.tsx

```diff
@@ -43,6 +43,7 @@ import useDebounce from '@tools/hooks/useDebounce'; // Debounce hook
 import { Column } from 'react-table';
 import { formatDateVerbose, formatFullName } from '@lib/utils/format'; // Verbose date formatter util
 import GenerateReportModal from '@components/admin/requests/reports/GenerateModal'; // Generate report modal
+import EmptyStateComponent from '@components/EmptyMessage';
```

Comment:
Could you please change all imported names for this component to `EmptyMessage` now?